### ISSUE TITLE
RunAlPipeline error: DockerDo : hcs::CreateComputeSystem ____ : The request is  not supported

### ISSUE THREAD

**Issue by FabioC34** [microsoft/navcontainerhelper#3419]
PLEASE DO NOT INCLUDE ANY PASSWORDS OR TOKENS IN YOUR ISSUE!!!

**Describe the issue**
Hi Freddy, since this morning my pipelines on Azure DevOps don't run anymore (yesterday and before everything worked flawlessly).
During Container Creation the pipeline give the error "DockerDo : hcs::CreateComputeSystem ____ : The request is  not supported"

**Full output of scripts**
```
Starting: Run Pipeline
==============================================================================
Task         : PowerShell
Description  : Run a PowerShell script on Linux, macOS, or Windows
Version      : 2.236.2
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/powershell
==============================================================================
Generating script.
Formatted command: . 'C:\vsts-agent-win-x64-3.225.0\_work\1\s\DevOps\scripts\DevOps-Pipeline.ps1' -environment "AzureDevOps" -version "ci" -appBuild $(Get-Date -Format yyyyMM) -appRevision 2660
========================== Starting Command Output ===========================
"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'C:\vsts-agent-win-x64-3.225.0\_work\_temp\a0de8c47-e112-46eb-9c9f-3f764e681885.ps1'"
Set artifact = /OnPrem/22.9.62830.0/it
Set pipelineName = Univergomma-ci
Set containerName = srvbc-test-univergomma-ci
Set installApps = ''
Set previousApps = ''
Set appSourceCopMandatoryAffixes = ''
Set appSourceCopSupportedCountries = ''
Set appFolders = '.'
Set testFolders = ''
Set memoryLimit = '8G'
Set additionalCountries = ''
Set genericImageName = ''
Set vaultNameForLocal = 'BuildVariables'
Set bcContainerHelperVersion = 'latest'
Set installTestFramework = False
Set installTestLibraries = False
Set installPerformanceToolkit = False
Set enableCodeCop = False
Set enableAppSourceCop = False
Set enablePerTenantExtensionCop = False
Set enableUICop = True
Set doNotSignApps = True
Set doNotRunTests = True
Set cacheImage = True
Set CreateRuntimePackages = True
BcContainerHelper version 6.0.9
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Running on Windows, PowerShell 5.1.20348.2227
Determining artifacts to use
  _____                               _
 |  __ \                             | |
 | |__) |_ _ _ __ __ _ _ __ ___   ___| |_ ___ _ __ ___
 |  ___/ _` | '__/ _` | '_ ` _ \ / _ \ __/ _ \ '__/ __|
 | |  | (_| | | | (_| | | | | | |  __/ |_  __/ |  \__ \
 |_|   \__,_|_|  \__,_|_| |_| |_|\___|\__\___|_|  |___/

Pipeline name                   Univergomma-ci
Container name                  srvbc-test-univergomma-ci
Image name                      bcimage
ArtifactUrl                     https://bcartifacts.azureedge.net/onprem/22.9.62830.0/it
SasToken                        Not Specified
BcAuthContext                   Not Specified
Environment                     
ReUseContainer                  False
KeepContainer                   False
useCompilerFolder               False
artifactCachePath               
useDevEndpoint                  False
Auth                            UserPassword
CompanyName                     
MemoryLimit                     8G
FailOn                          none
TreatTestFailuresAsWarnings     False
Enable Task Scheduler           False
Assign Premium Plan             False
Install Test Runner             False
Install Test Framework          False
Install Test Libraries          False
Install Perf. Toolkit           False
InstallOnlyReferencedApps       False
generateDependencyArtifact      False
CopySymbolsFromContainer        False
enableCodeCop                   False
enableAppSourceCop              False
enableUICop                     True
enablePerTenantExtensionCop     False
enableCodeAnalyzersOnTestApps   False
doNotPerformUpgrade             False
doNotPublishApps                False
uninstallRemovedApps            False
escapeFromCops                  False
doNotBuildTests                 False
doNotRunTests                   True
doNotRunBcptTests               False
useDefaultAppSourceRuleSet      False
rulesetFile                     
generateErrorLog                False
enableExternalRulesets          False
azureDevOps                     True
gitLab                          False
gitHubActions                   False
vsixFile                        
License file                    Not specified
CodeSignCertPfxFile             Not specified
CodeSignCertPfxPassword         Not specified
CodeSignCertIsSelfSigned        False
KeyVaultCertPfxFile             Not specified
KeyVaultCertPfxPassword         Not specified
KeyVaultClientId                
BuildOutputFile                 
ContainerEventLogFile           
TestResultsFile                 C:\vsts-agent-win-x64-3.225.0\_work\1\s\TestResults.xml
BcptTestResultsFile             C:\vsts-agent-win-x64-3.225.0\_work\1\s\bcptTestResults.json
TestResultsFormat               JUnit
AdditionalCountries             
PackagesFolder                  C:\vsts-agent-win-x64-3.225.0\_work\1\s\.packages
OutputFolder                    C:\vsts-agent-win-x64-3.225.0\_work\1\s\.output
BuildArtifactFolder             C:\vsts-agent-win-x64-3.225.0\_work\1\a
CreateRuntimePackages           True
AppVersion                      
AppBuild                        202403
AppRevision                     2660
SourceRepositoryUrl             
SourceCommit                    
BuildBy                         BcContainerHelper,6.0.9
BuildUrl                        
Install Apps
- None
Install Test Apps
- None
Previous Apps
- None
Application folders
- C:\vsts-agent-win-x64-3.225.0\_work\1\s
Test application folders
- None
BCPT Test application folders
- None
BCPT Test suites
- None
Custom CodeCops
- None

  _____       _ _ _                                          _        _
 |  __ \     | | (_)                                        (_)      (_)
 | |__) |   _| | |_ _ __   __ _    __ _  ___ _ __   ___ _ __ _  ___   _ _ __ ___   __ _  __ _  ___
 |  ___/ | | | | | | '_ \ / _` |  / _` |/ _ \ '_ \ / _ \ '__| |/ __| | | '_ ` _ \ / _` |/ _` |/ _ \
 | |   | |_| | | | | | | | (_| | | (_| |  __/ | | |  __/ |  | | (__  | | | | | | | (_| | (_| |  __/
 |_|    \__,_|_|_|_|_| |_|\__, |  \__, |\___|_| |_|\___|_|  |_|\___| |_|_| |_| |_|\__,_|\__, |\___|
                           __/ |   __/ |                                                 __/ |
                          |___/   |___/                                                 |___/

Pulling mcr.microsoft.com/businesscentral:ltsc2022

Pulling generic image took 0 seconds

   _____                _   _                               _        _
  / ____|              | | (_)                             | |      (_)
 | |     _ __ ___  __ _| |_ _ _ __   __ _    ___ ___  _ __ | |_ __ _ _ _ __   ___ _ __
 | |    | '__/ _ \/ _` | __| | '_ \ / _` |  / __/ _ \| '_ \| __/ _` | | '_ \ / _ \ '__|
 | |____| | |  __/ (_| | |_| | | | | (_| | | (__ (_) | | | | |_ (_| | | | | |  __/ |
  \_____|_|  \___|\__,_|\__|_|_| |_|\__, |  \___\___/|_| |_|\__\__,_|_|_| |_|\___|_|
                                     __/ |
                                    |___/

Creaing docker container
WARNING: Container name should not exceed 15 characters
BcContainerHelper is version 6.0.9
BcContainerHelper is running as administrator
HyperV is Disabled
UsePsSession is True
Host is Microsoft Windows Server 2022 Datacenter - 10.0.20348.2227
Docker Client Version is 24.0.5
Docker Server Version is 24.0.5
Removing Desktop shortcuts
Fetching all docker images
Fetching all docker volumes
ArtifactUrl and ImageName specified
Building image bcimage:onprem-22.9.62830.0-it based on mcr.microsoft.com/businesscentral:ltsc2022 with https://bcartifacts.azureedge.net/onprem/22.9.62830.0/it
Pulling latest image mcr.microsoft.com/businesscentral:ltsc2022
ltsc2022: Pulling from businesscentral
Digest: sha256:7473a1ca0616569f43a7b5e14add41f0a5fa8b4b5077abe86c5e4ade1af40d51
Status: Image is up to date for mcr.microsoft.com/businesscentral:ltsc2022
Generic Tag: 1.0.2.17
Container OS Version: 10.0.20348.2340 (ltsc2022)
Host OS Version: 10.0.20348.2227 (ltsc2022)
Using hyperv isolation
Files in c:\bcartifacts.cache\1gv0mgkh.5fs\my:
Copying Platform Artifacts
c:\bcartifacts.cache\onprem\22.9.62830.0\platform
Copying Database
Copying Licensefile
Copying Applications
C:\bcartifacts.cache\onprem\22.9.62830.0\it\Applications
c:\bcartifacts.cache\1gv0mgkh.5fs
Sending build context to Docker daemon  1.928GB

Step 1/6 : FROM mcr.microsoft.com/businesscentral:ltsc2022
 ---> c4cd601a3c1b
Step 2/6 : ENV DatabaseServer=localhost DatabaseInstance=SQLEXPRESS DatabaseName=CRONUS IsBcSandbox=N artifactUrl=https://bcartifacts.azureedge.net/onprem/22.9.62830.0/it filesOnly=False
 ---> Running in 63d1dfb8f0ac
Removing intermediate container 63d1dfb8f0ac
 ---> a3b63be4801b
Step 3/6 : COPY my /run/
 ---> e759101b13dd
Step 4/6 : COPY NAVDVD /NAVDVD/
 ---> c253cc1773ea
Step 5/6 : RUN \Run\start.ps1 -installOnly
 ---> Running in d6bda104ef50
New-Bcimage Telemetry Correlation Id: dd39ad1d-bca6-482a-b9f1-ae2a38212392
New-BcContainer Telemetry Correlation Id: 677558db-435d-4a52-9ecc-c6e20d815191
Removing Desktop shortcuts
Run-AlPipeline Telemetry Correlation Id: 1a60755d-68b4-4127-aefa-f15cedf975fe
##[error]DockerDo : hcs::CreateComputeSystem d6bda104ef501c53116f06e87f6e91322f7f694d54ab6940e6b72bf0b2f4fc2e: The request is 
not supported.
ExitCode: 1
Commandline: docker build --isolation=hyperv --memory 8G --no-cache --tag bcimage:onprem-22.9.62830.0-it 
c:\bcartifacts.cache\1gv0mgkh.5fs
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\6.0.9\ContainerHandling\New-NavImage.ps1:715 char:27
+ ...       if (!(DockerDo -command build -parameters @("--isolation=$isola ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,DockerDo
##[error]PowerShell exited with code '1'.
Finishing: Run Pipeline
```

Thank you for your support
Best Regards
Fabio

**Comment by freddydk** [comment]
Just shipped ContainerHelper 6.0.10 this morning, which should fix this issue.

**Comment by FabioC34** [comment]
Hi Freddy,
I tried again but unfortunately it doesn't work yet (I also set bcContainerHelperVersion = preview but didn't help)

Fabio

```
2024-03-14T10:57:32.9796789Z ##[section]Starting: Run Pipeline
2024-03-14T10:57:32.9960444Z ==============================================================================
2024-03-14T10:57:32.9960582Z Task         : PowerShell
2024-03-14T10:57:32.9960635Z Description  : Run a PowerShell script on Linux, macOS, or Windows
2024-03-14T10:57:32.9960715Z Version      : 2.236.2
2024-03-14T10:57:32.9960770Z Author       : Microsoft Corporation
2024-03-14T10:57:32.9960829Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/powershell
2024-03-14T10:57:32.9960900Z ==============================================================================
2024-03-14T10:57:34.2872086Z Generating script.
2024-03-14T10:57:34.3816259Z Formatted command: . 'C:\vsts-agent-win-x64-3.225.0\_work\1\s\DevOps\scripts\DevOps-Pipeline.ps1' -environment "AzureDevOps" -version "ci" -appBuild $(Get-Date -Format yyyyMM) -appRevision 2661
2024-03-14T10:57:34.3951190Z ========================== Starting Command Output ===========================
2024-03-14T10:57:34.4156775Z ##[command]"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'C:\vsts-agent-win-x64-3.225.0\_work\_temp\3c3a6947-d8e2-485a-a009-790cb06dad84.ps1'"
2024-03-14T10:57:34.9754729Z Set artifact = /OnPrem/22.9.62830.0/it
2024-03-14T10:57:34.9762308Z Set pipelineName = Univergomma-ci
2024-03-14T10:57:34.9799087Z Set containerName = srvbc-test-univergomma-ci
2024-03-14T10:57:35.0121904Z Set installApps = ''
2024-03-14T10:57:35.0148433Z Set previousApps = ''
2024-03-14T10:57:35.0158075Z Set appSourceCopMandatoryAffixes = ''
2024-03-14T10:57:35.0167035Z Set appSourceCopSupportedCountries = ''
2024-03-14T10:57:35.0194135Z Set appFolders = '.'
2024-03-14T10:57:35.0221017Z Set testFolders = ''
2024-03-14T10:57:35.0248192Z Set memoryLimit = '8G'
2024-03-14T10:57:35.0274961Z Set additionalCountries = ''
2024-03-14T10:57:35.0284477Z Set genericImageName = ''
2024-03-14T10:57:35.0311603Z Set vaultNameForLocal = 'BuildVariables'
2024-03-14T10:57:35.0338621Z Set bcContainerHelperVersion = 'preview'
2024-03-14T10:57:35.0431030Z Set installTestFramework = False
2024-03-14T10:57:35.0464586Z Set installTestLibraries = False
2024-03-14T10:57:35.0490956Z Set installPerformanceToolkit = False
2024-03-14T10:57:35.0518725Z Set enableCodeCop = False
2024-03-14T10:57:35.0542264Z Set enableAppSourceCop = False
2024-03-14T10:57:35.0568015Z Set enablePerTenantExtensionCop = False
2024-03-14T10:57:35.0594503Z Set enableUICop = True
2024-03-14T10:57:35.0621219Z Set doNotSignApps = True
2024-03-14T10:57:35.0647894Z Set doNotRunTests = True
2024-03-14T10:57:35.0675676Z Set cacheImage = True
2024-03-14T10:57:35.0702416Z Set CreateRuntimePackages = True
2024-03-14T10:57:35.7859743Z BcContainerHelper version 6.0.10
2024-03-14T10:57:36.4961976Z BC.HelperFunctions emits usage statistics telemetry to Microsoft
2024-03-14T10:57:36.8743465Z Running on Windows, PowerShell 5.1.20348.2227
2024-03-14T10:57:43.9756905Z Determining artifacts to use
2024-03-14T10:57:44.2474213Z   _____                               _
2024-03-14T10:57:44.2480268Z  |  __ \                             | |
2024-03-14T10:57:44.2486939Z  | |__) |_ _ _ __ __ _ _ __ ___   ___| |_ ___ _ __ ___
2024-03-14T10:57:44.2492237Z  |  ___/ _` | '__/ _` | '_ ` _ \ / _ \ __/ _ \ '__/ __|
2024-03-14T10:57:44.2498184Z  | |  | (_| | | | (_| | | | | | |  __/ |_  __/ |  \__ \
2024-03-14T10:57:44.2502622Z  |_|   \__,_|_|  \__,_|_| |_| |_|\___|\__\___|_|  |___/
2024-03-14T10:57:44.2507216Z 
2024-03-14T10:57:44.2513716Z Pipeline name                   Univergomma-ci
2024-03-14T10:57:44.2520296Z Container name                  srvbc-test-univergomma-ci
2024-03-14T10:57:44.2524651Z Image name                      bcimage
2024-03-14T10:57:44.2530528Z ArtifactUrl                     https://bcartifacts.azureedge.net/onprem/22.9.62830.0/it
2024-03-14T10:57:44.2547230Z SasToken                        Not Specified
2024-03-14T10:57:44.2559145Z BcAuthContext                   Not Specified
2024-03-14T10:57:44.2569078Z Environment                     
2024-03-14T10:57:44.2580988Z ReUseContainer                  False
2024-03-14T10:57:44.2593143Z KeepContainer                   False
2024-03-14T10:57:44.2604511Z useCompilerFolder               False
2024-03-14T10:57:44.2614132Z artifactCachePath               
2024-03-14T10:57:44.2625780Z useDevEndpoint                  False
2024-03-14T10:57:44.2637685Z Auth                            UserPassword
2024-03-14T10:57:44.2647043Z CompanyName                     
2024-03-14T10:57:44.2658459Z MemoryLimit                     8G
2024-03-14T10:57:44.2669231Z FailOn                          none
2024-03-14T10:57:44.2682371Z TreatTestFailuresAsWarnings     False
2024-03-14T10:57:44.2694436Z Enable Task Scheduler           False
2024-03-14T10:57:44.2707238Z Assign Premium Plan             False
2024-03-14T10:57:44.2719587Z Install Test Runner             False
2024-03-14T10:57:44.2731631Z Install Test Framework          False
2024-03-14T10:57:44.2742634Z Install Test Libraries          False
2024-03-14T10:57:44.2754114Z Install Perf. Toolkit           False
2024-03-14T10:57:44.2765919Z InstallOnlyReferencedApps       False
2024-03-14T10:57:44.2777537Z generateDependencyArtifact      False
2024-03-14T10:57:44.2788921Z CopySymbolsFromContainer        False
2024-03-14T10:57:44.2800876Z enableCodeCop                   False
2024-03-14T10:57:44.2812851Z enableAppSourceCop              False
2024-03-14T10:57:44.2824972Z enableUICop                     True
2024-03-14T10:57:44.2836537Z enablePerTenantExtensionCop     False
2024-03-14T10:57:44.2847878Z enableCodeAnalyzersOnTestApps   False
2024-03-14T10:57:44.2859997Z doNotPerformUpgrade             False
2024-03-14T10:57:44.2871320Z doNotPublishApps                False
2024-03-14T10:57:44.2884388Z uninstallRemovedApps            False
2024-03-14T10:57:44.2895789Z escapeFromCops                  False
2024-03-14T10:57:44.2907371Z doNotBuildTests                 False
2024-03-14T10:57:44.2919234Z doNotRunTests                   True
2024-03-14T10:57:44.2931305Z doNotRunBcptTests               False
2024-03-14T10:57:44.2943418Z useDefaultAppSourceRuleSet      False
2024-03-14T10:57:44.2951822Z rulesetFile                     
2024-03-14T10:57:44.2964336Z generateErrorLog                False
2024-03-14T10:57:44.2976919Z enableExternalRulesets          False
2024-03-14T10:57:44.2989090Z azureDevOps                     True
2024-03-14T10:57:44.3001375Z gitLab                          False
2024-03-14T10:57:44.3013673Z gitHubActions                   False
2024-03-14T10:57:44.3023253Z vsixFile                        
2024-03-14T10:57:44.3037380Z License file                    Not specified
2024-03-14T10:57:44.3044885Z CodeSignCertPfxFile             Not specified
2024-03-14T10:57:44.3063764Z CodeSignCertPfxPassword         Not specified
2024-03-14T10:57:44.3089032Z CodeSignCertIsSelfSigned        False
2024-03-14T10:57:44.3096589Z KeyVaultCertPfxFile             Not specified
2024-03-14T10:57:44.3103926Z KeyVaultCertPfxPassword         Not specified
2024-03-14T10:57:44.3113384Z KeyVaultClientId                
2024-03-14T10:57:44.3122273Z BuildOutputFile                 
2024-03-14T10:57:44.3131380Z ContainerEventLogFile           
2024-03-14T10:57:44.3142342Z TestResultsFile                 C:\vsts-agent-win-x64-3.225.0\_work\1\s\TestResults.xml
2024-03-14T10:57:44.3153082Z BcptTestResultsFile             C:\vsts-agent-win-x64-3.225.0\_work\1\s\bcptTestResults.json
2024-03-14T10:57:44.3163407Z TestResultsFormat               JUnit
2024-03-14T10:57:44.3191132Z AdditionalCountries             
2024-03-14T10:57:44.3202576Z PackagesFolder                  C:\vsts-agent-win-x64-3.225.0\_work\1\s\.packages
2024-03-14T10:57:44.3213540Z OutputFolder                    C:\vsts-agent-win-x64-3.225.0\_work\1\s\.output
2024-03-14T10:57:44.3225075Z BuildArtifactFolder             C:\vsts-agent-win-x64-3.225.0\_work\1\a
2024-03-14T10:57:44.3236534Z CreateRuntimePackages           True
2024-03-14T10:57:44.3245793Z AppVersion                      
2024-03-14T10:57:44.3257497Z AppBuild                        202403
2024-03-14T10:57:44.3269145Z AppRevision                     2661
2024-03-14T10:57:44.3278294Z SourceRepositoryUrl             
2024-03-14T10:57:44.3287871Z SourceCommit                    
2024-03-14T10:57:44.3299539Z BuildBy                         BcContainerHelper,6.0.10
2024-03-14T10:57:44.3308729Z BuildUrl                        
2024-03-14T10:57:44.3314189Z Install Apps
2024-03-14T10:57:44.3319875Z - None
2024-03-14T10:57:44.3325707Z Install Test Apps
2024-03-14T10:57:44.3331062Z - None
2024-03-14T10:57:44.3336473Z Previous Apps
2024-03-14T10:57:44.3342120Z - None
2024-03-14T10:57:44.3347664Z Application folders
2024-03-14T10:57:44.3398153Z - C:\vsts-agent-win-x64-3.225.0\_work\1\s
2024-03-14T10:57:44.3403644Z Test application folders
2024-03-14T10:57:44.3409301Z - None
2024-03-14T10:57:44.3414928Z BCPT Test application folders
2024-03-14T10:57:44.3420669Z - None
2024-03-14T10:57:44.3426407Z BCPT Test suites
2024-03-14T10:57:44.3432093Z - None
2024-03-14T10:57:44.3438247Z Custom CodeCops
2024-03-14T10:57:44.3443025Z - None
2024-03-14T10:57:44.3794371Z 
2024-03-14T10:57:44.3800638Z   _____       _ _ _                                          _        _
2024-03-14T10:57:44.3806724Z  |  __ \     | | (_)                                        (_)      (_)
2024-03-14T10:57:44.3812474Z  | |__) |   _| | |_ _ __   __ _    __ _  ___ _ __   ___ _ __ _  ___   _ _ __ ___   __ _  __ _  ___
2024-03-14T10:57:44.3819042Z  |  ___/ | | | | | | '_ \ / _` |  / _` |/ _ \ '_ \ / _ \ '__| |/ __| | | '_ ` _ \ / _` |/ _` |/ _ \
2024-03-14T10:57:44.3824856Z  | |   | |_| | | | | | | | (_| | | (_| |  __/ | | |  __/ |  | | (__  | | | | | | | (_| | (_| |  __/
2024-03-14T10:57:44.3830844Z  |_|    \__,_|_|_|_|_| |_|\__, |  \__, |\___|_| |_|\___|_|  |_|\___| |_|_| |_| |_|\__,_|\__, |\___|
2024-03-14T10:57:44.3837108Z                            __/ |   __/ |                                                 __/ |
2024-03-14T10:57:44.3843397Z                           |___/   |___/                                                 |___/
2024-03-14T10:57:44.3846718Z 
2024-03-14T10:57:44.5578283Z Pulling mcr.microsoft.com/businesscentral:ltsc2022
2024-03-14T10:57:44.8186618Z 
2024-03-14T10:57:44.8193396Z Pulling generic image took 0 seconds
2024-03-14T10:57:44.8498789Z 
2024-03-14T10:57:44.8505524Z    _____                _   _                               _        _
2024-03-14T10:57:44.8512141Z   / ____|              | | (_)                             | |      (_)
2024-03-14T10:57:44.8518692Z  | |     _ __ ___  __ _| |_ _ _ __   __ _    ___ ___  _ __ | |_ __ _ _ _ __   ___ _ __
2024-03-14T10:57:44.8524741Z  | |    | '__/ _ \/ _` | __| | '_ \ / _` |  / __/ _ \| '_ \| __/ _` | | '_ \ / _ \ '__|
2024-03-14T10:57:44.8530023Z  | |____| | |  __/ (_| | |_| | | | | (_| | | (__ (_) | | | | |_ (_| | | | | |  __/ |
2024-03-14T10:57:44.8535656Z   \_____|_|  \___|\__,_|\__|_|_| |_|\__, |  \___\___/|_| |_|\__\__,_|_|_| |_|\___|_|
2024-03-14T10:57:44.8541071Z                                      __/ |
2024-03-14T10:57:44.8546603Z                                     |___/
2024-03-14T10:57:44.8549637Z 
2024-03-14T10:57:44.9484353Z Creaing docker container
2024-03-14T10:57:45.0495374Z WARNING: Container name should not exceed 15 characters
2024-03-14T10:57:45.1140144Z BcContainerHelper is version 6.0.10
2024-03-14T10:57:45.1146162Z BcContainerHelper is running as administrator
2024-03-14T10:57:46.1275345Z HyperV is Disabled
2024-03-14T10:57:46.1295096Z UsePsSession is True
2024-03-14T10:57:46.1312339Z Host is Microsoft Windows Server 2022 Datacenter - 10.0.20348.2227
2024-03-14T10:57:46.1758339Z Docker Client Version is 24.0.5
2024-03-14T10:57:46.1765841Z Docker Server Version is 24.0.5
2024-03-14T10:57:46.3024422Z Removing Desktop shortcuts
2024-03-14T10:57:47.5215814Z Fetching all docker images
2024-03-14T10:57:47.5651835Z Fetching all docker volumes
2024-03-14T10:57:47.6065771Z ArtifactUrl and ImageName specified
2024-03-14T10:57:48.2389619Z Building image bcimage:onprem-22.9.62830.0-it based on mcr.microsoft.com/businesscentral:ltsc2022 with https://bcartifacts.azureedge.net/onprem/22.9.62830.0/it
2024-03-14T10:57:48.2395767Z Pulling latest image mcr.microsoft.com/businesscentral:ltsc2022
2024-03-14T10:57:48.4788078Z ltsc2022: Pulling from businesscentral
2024-03-14T10:57:48.4802162Z Digest: sha256:7473a1ca0616569f43a7b5e14add41f0a5fa8b4b5077abe86c5e4ade1af40d51
2024-03-14T10:57:48.4807098Z Status: Image is up to date for mcr.microsoft.com/businesscentral:ltsc2022
2024-03-14T10:57:48.5377345Z Generic Tag: 1.0.2.17
2024-03-14T10:57:48.5840896Z Container OS Version: 10.0.20348.2340 (ltsc2022)
2024-03-14T10:57:48.5848154Z Host OS Version: 10.0.20348.2227 (ltsc2022)
2024-03-14T10:57:48.5906202Z Using hyperv isolation
2024-03-14T10:57:48.5990415Z Files in c:\bcartifacts.cache\v4m50zup.23h\my:
2024-03-14T10:57:48.6039023Z Copying Platform Artifacts
2024-03-14T10:57:48.6087232Z c:\bcartifacts.cache\onprem\22.9.62830.0\platform
2024-03-14T10:57:53.9302369Z Copying Database
2024-03-14T10:57:54.1890809Z Copying Licensefile
2024-03-14T10:57:54.4510463Z Copying Applications
2024-03-14T10:57:54.4518742Z C:\bcartifacts.cache\onprem\22.9.62830.0\it\Applications
2024-03-14T10:57:59.5138285Z c:\bcartifacts.cache\v4m50zup.23h
2024-03-14T10:58:28.0812970Z Sending build context to Docker daemon  1.928GB
2024-03-14T10:58:28.0816580Z 
2024-03-14T10:58:28.2490597Z Step 1/6 : FROM mcr.microsoft.com/businesscentral:ltsc2022
2024-03-14T10:58:28.2504528Z  ---> c4cd601a3c1b
2024-03-14T10:58:28.2510499Z Step 2/6 : ENV DatabaseServer=localhost DatabaseInstance=SQLEXPRESS DatabaseName=CRONUS IsBcSandbox=N artifactUrl=https://bcartifacts.azureedge.net/onprem/22.9.62830.0/it filesOnly=False
2024-03-14T10:58:28.4052660Z  ---> Running in b967801a97ac
2024-03-14T10:58:28.7704862Z Removing intermediate container b967801a97ac
2024-03-14T10:58:28.7713874Z  ---> 795f9494983b
2024-03-14T10:58:28.7720549Z Step 3/6 : COPY my /run/
2024-03-14T10:58:29.5180252Z  ---> 4ed9901db09e
2024-03-14T10:58:29.5190817Z Step 4/6 : COPY NAVDVD /NAVDVD/
2024-03-14T10:59:17.1328888Z  ---> efcc2eaf53b3
2024-03-14T10:59:17.1335002Z Step 5/6 : RUN \Run\start.ps1 -installOnly
2024-03-14T10:59:17.2755940Z  ---> Running in 15e4b97a8836
2024-03-14T10:59:21.8741435Z New-Bcimage Telemetry Correlation Id: 143acf51-211f-420f-a574-2f255de93814
2024-03-14T10:59:22.3896149Z New-BcContainer Telemetry Correlation Id: 9adf7362-9734-4f99-b5f2-d9d7e82a4776
2024-03-14T10:59:22.5224290Z Removing Desktop shortcuts
2024-03-14T10:59:23.0822510Z Run-AlPipeline Telemetry Correlation Id: 1674bb83-1602-41fe-91bf-582874ba71e5
2024-03-14T10:59:23.2395248Z ##[error]DockerDo : hcs::CreateComputeSystem 15e4b97a8836a40affed069340b669995a0f51db56f9cedf2f9c5d310d3cfcba: The request is 
not supported.
ExitCode: 1
Commandline: docker build --isolation=hyperv --memory 8G --no-cache --tag bcimage:onprem-22.9.62830.0-it 
c:\bcartifacts.cache\v4m50zup.23h
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\6.0.10\ContainerHandling\New-NavImage.ps1:715 char:27
+ ...       if (!(DockerDo -command build -parameters @("--isolation=$isola ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,DockerDo
2024-03-14T10:59:23.2443057Z ##[error]PowerShell exited with code '1'.
2024-03-14T10:59:23.3375877Z ##[section]Finishing: Run Pipeline

```

**Comment by freddydk** [comment]
This should be fixed in 6.0.11
